lib/parity/environment.rb

```diff
@@ -37,7 +41,9 @@ def backup
     end
 
     def deploy
-      if deploy_to_heroku && run_migration?
+      pending_migrations = run_migration?
+
+      if deploy_to_heroku && pending_migrations
         migrate
       end
```

Comment:
Actually, I think the default `|| false` will cause this method to exit with a non-zero status code if there's no migration to run.